include/deal.II/dofs/dof_renumbering.h

```diff
@@ -1273,6 +1273,35 @@ namespace DoFRenumbering
     std::vector<types::global_dof_index> &new_dof_indices,
     const DoFHandler<dim, spacedim>      &dof_handler);
 
+  /**
+   * Reorder the degrees of freedom in lexicographic order of support points.
+   *  The tolerance used to determine whether two support points are equal.
+   *
+   * This is particularly useful for finite element spaces that are defined on
+   * tensor product grids, such as the ones generated by
+   * GridGenerator::subdivided_hyper_rectangle() or GridGenerator::hyper_cube().
```

Comment:
I would suggest to use the same form for both rectangles and cubes, since both also have the subdivided variants:
```suggestion
   * GridGenerator::hyper_rectangle() or GridGenerator::hyper_cube() or
   * their subdivided variants.
```